### PR TITLE
Pricing page

### PR DESCRIPTION
**Pull request by jonoroboto**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Screenshots (optional)**

**Checklist**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by jonoroboto** [comment]
Don't green light yet, still need to wire some elements up correctly

File: packages/www/pages/pricing/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Layout from "layouts/main";
+import { Box, Heading, Text, Container } from "@livepeer/design-system";
+// import Prefooter from "components/Site/Prefooter";
+// import PricingCalculator from "components/Site/Pricing/pricingCalculator";
```
Comment (clacladev) [inline packages/www/pages/pricing/index.tsx]: Remove the commented import to keep it clean

**Review by clacladev — COMMENTED** [review]
When opening the Pricing page there is a warning in console that would be nice to be addressed:
`Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.`

It was probably not introduced by this PR, but it's still something that we want to address so to it makes it easier to keep updated with the latest Next.js versions

File: packages/www/pages/pricing/index.tsx (hunk 1 above)
Comment (jonoroboto) [reply packages/www/pages/pricing/index.tsx]: Solved this, and removed excess imports

**Comment by jonoroboto** [comment]
> When opening the Pricing page there is a warning in console that would be nice to be addressed: `Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://reactjs.org/link/unsafe-component-lifecycles for details.`
> 
> It was probably not introduced by this PR, but it's still something that we want to address so to it makes it easier to keep updated with the latest Next.js versions

Just be aware this is coming from React Reveal, going to try and update, but if that doesn't work will need to assess whether console issue is more or less important than fade animations.

**Comment by jonoroboto** [comment]
Also just cleared all console logs, shouldn't have any additional issues, notice rehydration issue related to next link

**Review by clacladev — APPROVED** [review]
Great job! Ready to go :D

**Pull request merged**